cockatrice/src/dialogs/dlg_settings.cpp

```diff
@@ -1290,6 +1294,7 @@ void MessagesSettingsPage::retranslateUi()
     aAdd->setText(tr("Add New Message"));
     aEdit->setText(tr("Edit Message"));
     aRemove->setText(tr("Remove Message"));
+    localTimeCheckBox.setText(tr("Use Room Time over Real-Life Time"));
```

Comment:
I think a more user friendly approach would be to present this as a two option spinbox to users, eg:
```
Chat time stamps use: ->Local clock
                      - Match timer
```